src/Components/bookHub/BookCardsDisplay.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import Card from './BookCard';
+
+// airtable configuration
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+function CardsDisplay() {
```

Comment:
In future work this is a function you would want to rename too so that it is more clearly describing the function! 
```suggestion
function BookCardsDisplay() {
```
But you can pull all of this code to the Home page for the next sprint, so just keep connotative naming in mind as you move on!